### ISSUE TITLE
validated-non-primitive-property-needs-type-decorator errors on enum property

### ISSUE THREAD

**Issue by iddan-flycode** [darraghoriordan/eslint-plugin-nestjs-typed#16]
```typescript
enum Foo {
   BAR
}

// invalid
class Example
  @ApiProperty({ enum: Foo, enumName: 'Foo' })
  @Allow()
  format!: Foo;
}
```

**Comment by darraghoriordan** [comment]
fixed in 3.5.6

thanks for reporting
